Fresh clone of the repo errors in yarn start in workshop/01-fetchGraphql

Hello!

I'm using

- macOS Monterey 12.5
- node v16.20.1
- yarn 1.22.19
After cloning the repo, running `yarn` and then in the workshop/01-fetchGraphql `yarn start` I get:

```
01-fetchGraphql ❯ yarn start
yarn run v1.22.19
warning package.json: No license field
$ webpack serve --hot
Failed to load ./.env.
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:8444/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.31.218:8444/
<i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::1]:8444/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/macuser/Documents/work/relay-workshop/workshop/01-fetchGraphql/build' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
<i> [webpack-dev-middleware] wait until bundle finished: /
asset main.js 3.7 MiB [emitted] (name: main) 1 related asset
asset index.html 262 bytes [emitted]
orphan modules 652 KiB [orphan] 385 modules
runtime modules 28.6 KiB 15 modules
modules by path ../../node_modules/ 3.06 MiB 669 modules
modules by path ../../packages/ 44.1 KiB
  modules by path ../../packages/ui/src/*.tsx 16.6 KiB 10 modules
  modules by path ../../packages/route/src/*.tsx 27.4 KiB
    ../../packages/route/src/index.tsx 1.41 KiB [built] [code generated]
    + 9 modules
modules by path ./src/*.tsx 3.47 KiB
  ./src/index.tsx 1.32 KiB [built] [code generated]
  ./src/App.tsx 2.15 KiB [built] [code generated]
fs (ignored) 15 bytes [built] [code generated]
util (ignored) 15 bytes [built] [code generated]
util (ignored) 15 bytes [built] [code generated]

WARNING in ../../node_modules/react-router-config/esm/react-router-config.js 41:38-44
export 'Switch' (imported as 'Switch') was not found in 'react-router' (possible exports: AbortedDeferredError, Await, MemoryRouter, Navigate, NavigationType, Outlet, Route, Router, RouterProvider, Routes, UNS
AFE_DataRouterContext, UNSAFE_DataRouterStateContext, UNSAFE_LocationContext, UNSAFE_NavigationContext, UNSAFE_RouteContext, UNSAFE_mapRouteProperties, UNSAFE_useRouteId, UNSAFE_useRoutesImpl, createMemoryRout
er, createPath, createRoutesFromChildren, createRoutesFromElements, defer, generatePath, isRouteErrorResponse, json, matchPath, matchRoutes, parsePath, redirect, redirectDocument, renderMatches, resolvePath, u
nstable_useBlocker, useActionData, useAsyncError, useAsyncValue, useHref, useInRouterContext, useLoaderData, useLocation, useMatch, useMatches, useNavigate, useNavigation, useNavigationType, useOutlet, useOutl
etContext, useParams, useResolvedPath, useRevalidator, useRouteError, useRouteLoaderData, useRoutes)
 @ ../../packages/route/src/createRouter.tsx 3:0-50 8:24-35 88:22-33 95:22-33
 @ ../../packages/route/src/index.tsx 4:0-57 4:0-57
 @ ../../packages/ui/src/BackButton.tsx 7:0-45 12:18-28 25:10-20
 @ ../../packages/ui/src/index.tsx 10:0-53 10:0-53
 @ ./src/App.tsx 5:0-53 16:44-51 16:156-162 20:42-49 24:56-60 29:178-184 29:234-240
 @ ./src/index.tsx 6:0-24 12:46-49

ERROR in ../../node_modules/next/dist/compiled/gzip-size/index.js 1:2928-2943
Module not found: Error: Can't resolve 'zlib' in '/Users/macuser/Documents/work/relay-workshop/node_modules/next/dist/compiled/gzip-size'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "zlib": require.resolve("browserify-zlib") }'
        - install 'browserify-zlib'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "zlib": false }
```

Two issues here: ERROR and (possibly an issue) BREAKING CHANGE.

--- a/lerna.json
+++ b/lerna.json
@@ -1,6 +1,9 @@
 {
   "packages": [
-    "packages/*"
+    "packages/*",
+    "apps/*",
+    "workshop/*",
+    "solutions/*"
   ],
   "version": "1.0.0",
   "npmClient": "yarn"

--- a/package.json
+++ b/package.json
@@ -36,7 +36,9 @@
     "@types/supertest": "2.0.12",
     "@typescript-eslint/eslint-plugin": "5.62.0",
     "@typescript-eslint/parser": "5.62.0",
+    "assert": "^2.1.0",
     "babel-jest": "29.6.4",
+    "browserify-zlib": "^0.2.0",
     "concurrently": "^8.0.1",
     "crypto-browserify": "^3.12.0",
     "esbuild-register": "^3.3.2",
@@ -73,7 +75,8 @@
     "prettier": "3.0.3",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
-    "typescript": "5.2.2"
+    "typescript": "5.2.2",
+    "util": "^0.12.5"
   },
   "lint-staged": {
     "*.{js,ts,tsx}": [

